treerec/tests/testing_utils.slim

```diff
@@ -1,4 +1,4 @@
-// Note call to initializeTreeSeq at the end.
+""// Define various functions used in SLiM testing
```

Comment:
spurious `""` here?

Reply:
Oops. Sorry

Reply:
That file isn't used - it's an old named version that I forgot to delete. Will delete it and update.

Reply:
Done